hack/jenkins/job-configs/kubernetes-e2e.yaml

```diff
@@ -5,7 +5,8 @@
         daysToKeep: 7
     builders:
         - shell: |
-            curl -fsS --retry 3  "https://raw.githubusercontent.com/kubernetes/kubernetes/{branch}/hack/jenkins/e2e.sh" | bash -
+            curl -fsS --retry 3  "https://raw.githubusercontent.com/kubernetes/kubernetes/{branch}/hack/jenkins/e2e-jobs.sh" | bash -
```

Comment:
hm. do environment variables that get exported here persist into the next script?